src/main/java/backend/github/ApiQuotaInfo.java

```diff
@@ -0,0 +1,36 @@
+package backend.github;
+
+import util.Utility;
+
+/**
+ * Holds GitHub API quota information. Data are immutable.
+ */
+public class ApiQuotaInfo {
+
+    /**
+     * The number of API requests remaining in the current rate limit window.
+     */
+    public final int remainingRequests;
```

Comment:
no, since there's already getter methods for these variables. will take note in future, thanks!